src/auth.rs

```diff
@@ -0,0 +1,71 @@
+use std::io;
+
+use anyhow::{bail, format_err, Context};
+use serde::{Deserialize, Serialize};
+use toml_edit::Document;
+
+use crate::config::write_if_not_exists;
+use crate::home::Home;
+
+pub static MANIFEST_FILE_NAME: &str = "auth.toml";
```

Comment:
This should be a `const` instead of `static`. There isn't much functional difference in this case, but a `const` will be inlined wherever it's used, thus being more efficient (and idiomatic).

Reply:
You're right, although I find it much more common to use a `const` in a situation like this as opposed to `static`

Reply:
No need. Want to keep consistency with the rest of the project that uses this.